docs/guides/running-clustered-experiments.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 2
+---
+
+# Running Clustered Experiments
```

Comment:
I think this guide should be called `Analyzing Clustered Experiments`. The guide explains how to define analysis unit-level metrics in terms of randomization unit-level metrics. Running a clustered experiment requires much more, especially when clusters are not given to you for free (such as example 2 below) and the hard part is to build suitable clusters.

Reply:
Great point. Changed here: https://github.com/Eppo-exp/eppo-docs/commit/09661436891d9853ac23cfe1db1da1cf7def4403